packages/material-ui/src/Theme/MaterialUIContext.tsx

```diff
@@ -83,10 +99,25 @@ if (mui && Object.keys(mui).length && icons && Object.keys(icons).length) {
     Slider,
     TextField,
     Typography,
+    // icons
     AddIcon,
     ArrowDownwardIcon,
     ArrowUpwardIcon,
     ErrorIcon,
     RemoveIcon,
   };
 }
+
+export let DefaultChildren = () => {
+  if (MaterialUIContext.Box && MaterialUIContext.Button && MaterialUIContext.AddIcon) {
```

Comment:
Should we remove `AddIcon`?

Reply:
I'm guessing it is there to ensure that all of the icons were also loaded